### PR TITLE
Improve the API docs and MD files

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$title

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by daneshk — APPROVED** [review]

File: ballerina/Package.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The I/O library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
 </a> language.
 
-This package provides APIs for file input/output operations and the console input/output.
+This package provides APIs for file input/output operations, and the console input/output.
```
Comment (praneesha) [inline ballerina/Package.md:6]:
```suggestion
This package provides APIs for file input/output operations and the console input/output.
```

File: ballerina/file_string_io.bal [hunk 2]
```diff
@@ -131,8 +131,8 @@ public isolated function fileWriteJson(string path, json content) returns Error?
 # ```
 # + path - The path of the XML file
 # + content - XML content to write
-# + xmlOptions - XML writing options(XML entity type and DOCTYPE)
-# + fileWriteOption - file write option(`OVERWRITE` and `APPEND` are the possible values, and the default value is `OVERWRITE`)
+# + xmlOptions - XML writing options (XML entity type and DOCTYPE)
+# + fileWriteOption - File write option (`OVERWRITE` and `APPEND` are the possible values, and the default value is `OVERWRITE`)
```
Comment (praneesha) [inline ballerina/file_string_io.bal:135]:
```suggestion
# + fileWriteOption - File write option (`OVERWRITE` and `APPEND` are the possible values and the default value is `OVERWRITE`)
```

File: ballerina/readable_data_channel.bal [hunk 3]
```diff
@@ -83,7 +83,7 @@ public class ReadableDataChannel {
     # boolean|io:Error result = dataChannel.readBool();
     # ```
     #
-    # + return - boolean value which is read or else `io:Error` if any error occurred
+    # + return - Boolean value which is read or else `io:Error` if any error occurred
```
Comment (praneesha) [inline ballerina/readable_data_channel.bal:86]:
```suggestion
    # + return - Boolean value, which is read or else `io:Error` if any error occurred
```

Comment (BuddhiWathsala) [reply ballerina/readable_data_channel.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c

File: ballerina/file_string_io.bal (hunk 2 above)
Comment (BuddhiWathsala) [reply ballerina/file_string_io.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c

File: ballerina/Package.md (hunk 1 above)
Comment (BuddhiWathsala) [reply ballerina/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c

File: ballerina/print.bal [hunk 4]
```diff
@@ -29,14 +29,14 @@ public type FileOutputStream stdout|stderr;
...
-# + insertions - parameterized values/expressions after evaluations as an array
+# + strings - String values of the template as an array
+# + insertions - Parameterized values/expressions after evaluations as an array
 public type PrintableRawTemplate object {
     public string[] & readonly strings;
     public Printable[] insertions;
 };
 
-# Prints `any`, `error`, or string templates(such as `The respective int value is ${val}`) value(s) to the STDOUT.
+# Prints `any`, `error`, or string templates (such as `The respective int value is ${val}`) value(s) to the STDOUT.
```
Comment (praneesha) [inline ballerina/print.bal:39]:
```suggestion
# Prints `any`, `error`, or string templates (such as `The respective int value is ${val}`) value(s) to the `STDOUT`.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**